src/features/rewards/walletWrapper/index.tsx

```diff
@@ -71,18 +92,21 @@ export interface Props {
   grants?: Grant[]
   alert?: AlertWallet | null
   id?: string
-  gradientTop?: string
+  gradientTop?: string,
+  notification?: Notification
 }
 
 interface State {
-  grantDetails: boolean
+  grantDetails: boolean,
+  notificationDismissed: boolean
```

Comment:
We don't need dismiss state. We should only have onNotificationDismissed callback and that will be handled by client.

Reply:
@NejcZdovc removed from state, now uses callback